DistFiles/ReleaseNotes.md

```diff
@@ -1,15 +1,16 @@
 ﻿## Important Notices
 
-### Problems with Andika And other fonts
-- When you create a PDF using Andika, styles such as bold, italic etc. will be lost. This will happen with any font that doesn't have an have a real bold/italic/etc face. This is problem outside of our control at this time (we've reported it to Mozilla). SIL expects to release a new version of Andika in April which will fix this problem for most languages when using that font.
+### Andika Replaced By "Andika New Basic" 
+- Many Bloom books use SIL's free "Andika" literacy font. When you create a PDF using Andika, styles such as bold, italic etc. are be lost. This will happen with any font that doesn't have an have a real bold/italic/etc face. This is problem outside of our control at this time (we've reported it to Mozilla). Happily, SIL has released a subset of Andika named [Andika New Basic](http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=andika). Bloom now installs this font for you and uses it by default. If your language is not fully supported by Andika New Basic, please chose another font, idealy one which includes built-in bold and italic. You can test your font by using CTRL+B to make some text bold, then going to publish and looking to see if bold makes it through to the PDF.
```

Comment:
I think my release notes there were misleading. I've modified them to say "Many Bloom Collections" instead of books. See, most of our existing books don't specify a font, so the font selection actually comes from the language settings. Tested this by removing Andika and opening a book in an existing collection. Bloom gave a message about Andika being missing, exactly ones (as designed). In a future Commit, we'll do some migration, so that people won't even see that.